experimental/builder/test/unit_conv_tensor_layout.cpp

```diff
@@ -4,116 +4,481 @@
 #include <gtest/gtest.h>
 #include <type_traits>
 
-// Include the helper file we're testing
 #include "ck_tile/builder/factory/helpers/conv_tensor_layout.hpp"
+#include "impl/conv_signature_types.hpp"
 
 namespace {
 
 namespace ckb = ::ck_tile::builder;
+using ::ck_tile::builder::DataType;
+using ::ck_tile::builder::ElementwiseOperation;
+using ::ck_tile::builder::TensorLayout;
+using ::ck_tile::builder::factory::internal::AuxiliaryTensorLayouts;
 using ::ck_tile::builder::factory::internal::ConvTensorLayouts;
-using ::ck_tile::builder::factory::internal::GetTensorLayout;
+using ::ck_tile::builder::factory::internal::LayoutToCK;
+
+using namespace ::ck_tile::builder::test;
 using enum ::ck_tile::builder::ConvDirection;
```

Comment:
It's much cleaner to only have `using enum` in a function scope (even if it's repeated many times) rather than have it at file scope.